src/it/resources/org/checkstyle/suppressionxpathfilter/finalparameters/SuppressionXpathRegressionFinalParameters3.java

```diff
@@ -0,0 +1,7 @@
+package org.checkstyle.suppressionxpathfilter.finalparameters;
+
+public class SuppressionXpathRegressionFinalParameters3 {
+
+    public void method(String[] arg) { // warn
```

Comment:
This is kind of the same as SuppressionXpathRegressionFinalParameters1

Please update this file to report violation on method of anonymous class

Reply:
@romani done